handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/filter/IsolationDiscoveryFilter.java

```diff
@@ -92,8 +92,11 @@ public DiscoveryTreeNode discovery(DiscoveryContext context, DiscoveryTreeNode p
     }
 
     DiscoveryTreeNode child = new DiscoveryTreeNode();
-    if (filteredServers.isEmpty() && DynamicPropertyFactory.getInstance()
-        .getBooleanProperty("servicecomb.loadbalance.filter.isolation.emptyInstanceProtectionEnabled", false).get()) {
+    if (ZoneAwareDiscoveryFilter.GROUP_Instances_All
+        .equals(context.getContextParameter(ZoneAwareDiscoveryFilter.KEY_ZONE_AWARE_STEP)) && filteredServers.isEmpty()
+        && DynamicPropertyFactory.getInstance()
```

Comment:
move "servicecomb.loadbalance.filter.isolation.emptyInstanceProtectionEnabled" to OperationConfig

Reply:
ok, I will move it to the OperationConfig

Reply:
the latest version. set **servicecomb.loadbalance.filter.isolation.emptyInstanceProtectionEnabled** in class IsolationDiscoveryFilter#emptyProtection. do not move to **OperationConfig**